### PR TITLE
Updating MakeFile and fixing monologue memory parameter issue

### PR DESCRIPTION
**Pull request by risingsunomi**
Updates to makefile and monologue agent memory

- agenthub/monologue_agent/utils/memory.py was using base_url as parameter in call to llama_index.embeddings.openai.OpenAIEmbedding. This was causing an issue when setting the LLM embedding option to OpenAI but not the LLM base url. Also from LlamaIndex docs, that parameter is not needed. I have removed that and place a default model parameter of text-embedding-ada-002

- I have added more embedding model configuration settings in make file for setup-config. This now sets the azure or llama2 needed options if one of those embedding models are selected

### CODE REVIEW

**Comment by risingsunomi** [comment]
Checking the PR list more, think this is similar to https://github.com/OpenDevin/OpenDevin/pull/616 but I have added more logic depending on the embedding model chosen

File: Makefile [hunk 1]
```diff
@@ -49,6 +49,21 @@ setup-config:
...
+	 echo "LLM_EMBEDDING_MODEL=\"$$llm_embedding_model\"" >> $(CONFIG_FILE).tmp; \
+	 if [ "$$llm_embedding_model" = "llama2" ]; then \
+	 read -p "Enter the local model URL: " llm_base_url; \
+	 echo "LLM_BASE_URL=\"$$llm_base_url\"" >> $(CONFIG_FILE).tmp; \
+	 elif [ "$$llm_embedding_model" = "azureopenai" ]; then \
+	 read -p "Enter the Azure endpoint URL: " llm_base_url; \
+	 echo "LLM_BASE_URL=\"$$llm_base_url\"" >> $(CONFIG_FILE).tmp; \
+	 read -p "Enter the Azure LLM Deployment Name: " llm_deployment_name; \
+	 echo "LLM_DEPLOYMENT_NAME=\"$$llm_deployment_name\"" >> $(CONFIG_FILE).tmp; \
+	 read -p "Enter the Azure API Version: " llm_api_version; \
```
Comment (rbren) [inline Makefile:63]: possible to indent to make this more readable?

**Review by rbren — APPROVED** [review]

**Comment by rbren** [comment]
Thanks for the fix!

File: Makefile (hunk 1 above)
Comment (risingsunomi) [reply Makefile:63]: Yes will work on that and resubmit

Comment (risingsunomi) [reply Makefile]: Indent added. Let me know if that is okay.

**Pull request merged**